packages/widget/src/state/swapExecutionPage.ts

```diff
@@ -96,6 +97,7 @@ export const setSwapExecutionStateAtom = atom(null, (get, set) => {
       }
     },
     onTransactionBroadcast: async (txInfo) => {
+      setUser({ id: txInfo?.txHash });
```

Comment:
did we want to set this to the user address?

Reply:
yeah, we should keep both, one is setting id and one is setting username so both will be kept